HotSpot/DataManager.m

```diff
@@ -59,4 +59,23 @@ + (void)test {
     
 }
 
++ (Listing *)sampleListingForTestingWithCompletion:(void(^)(Listing *listing, NSError *error))completion{
+    PFGeoPoint *point = [PFGeoPoint geoPointWithLatitude:37.773972 longitude:-122.431297]; // san francisco
+    __block Listing *listing = nil;
+    [DataManager getListingsNearLocation:point withCompletion:^(NSArray<Listing *> * _Nonnull listings, NSError * _Nonnull error) {
+        completion(listings[0], error);
+    }];
+    return listing;
```

Comment:
this will always be nil! the return type should be void, since you're passing the listing through the completion block